ert3/console/_console.py

```diff
@@ -246,10 +252,13 @@ def _init(args: Any) -> None:
 def _build_local_test_run_config(
     experiment_run_config: ert3.config.ExperimentRunConfig,
     plugin_registry: ConfigPluginRegistry,
+    iens: int = 0,
 ) -> ert3.config.ExperimentRunConfig:
     """Validate and modify a run configuration for a local test run scenario.
 
-    The ensemble size is set to 1, and the driver is set to "local".
+    All realizations except the selected realization are set inactive.
```

Comment:
```suggestion
    All realizations except iens are set inactive.
```

Reply:
might suggest that `iens` is a bad name at this abstraction level

Reply:
Agree. I reluctantly used `iens` in this function, but now I have changed it to `realization`. Should be fair given this is a private function.